packages/gatsby-theme-api-docs/src/components/resource/method/request-response-examples.js

```diff
@@ -9,9 +9,9 @@ const RequestResponseExamples = (props) => {
   const apiTypes = useApiTypes();
   const responsesCodeExamples = [];
   if (props.responses) {
-    props.responses.forEach((response) => {
+    props.responses.forEach((response, index) => {
       const typeDisplayName =
-        response.body && response.body.applicationjson.type;
+        response.body && response.body[props.contentType[index]].type;
```

Comment:
what if this expression is undefined? `body[props.contentType[index]]`
Also, you can use the [conditional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) instead of repeating `response.body && ....`

So it would become `response?.body?.[props.contentType[index]]?.type`